sklearn/utils/multiclass.py

```diff
@@ -342,11 +342,27 @@ def _check_partial_fit_first_call(clf, classes=None):
 
         else:
             # This is the first call to partial_fit
+            if issparse(y):
+                raise ValueError("Sparse y is not supported.")
+
+            # Validate classes against unique labels in y
+            y_, classes_ = np.array(y), np.array(classes)
+            n_outputs = 1 if len(y_.shape) == 1 else y_.shape[1]
+            y_ = y_.reshape(y_.shape[0], n_outputs)
+            classes_ = classes_.reshape(n_outputs, classes_.shape[0])
+
+            for i in range(n_outputs):
+                unique_y = np.unique(y_[:, :i])
+                if not np.all(in1d(unique_y, classes_[i])):
+                     raise ValueError("The target label(s) %s in y do not exist in the "
+                                      "initial classes %s" % 
+                                      (unique_y[~unique_y_in_classes], 
```

Comment:
This variable no longer exists (which means that you should be catching the NameError somewhere?)